data-pipeline/src/pipeline/mappings.py

```diff
@@ -14,11 +14,14 @@ def map_ofsted_rating(rating: str):
             return rating
 
 
-def map_phase_type(establishment_code: int, phase_code: int, provision: str):
-
-    if (pd.isna(establishment_code) or pd.isna(phase_code) or pd.isna(provision)):
+def map_phase_type(
+    establishment_code: int,
+    phase_code: int,
+    provision: str | None = None,
```

Comment:
Discussed elsewhere but we'll need to validate this: the academies will only have the GIAS-sourced first two values so this will default to `None` in those instances.

Reply:
Do we know the impact across the 3 years?  (i.e. how schools in the AAR don't appear in the associated GIAS file gias, gias_all_link or gias_links)

Reply:
As far as I can tell, there's a single academy affected. Based on the academies-master-list I have, it's not actually present; based on the data in d01 and Josh, it's there and gets populated (which I think speaks to the wider reason that file needs to go).

I flagged "phases" as something specifically to look at during QA.